src/zkevm_specs/evm_circuit/execution/error_gas_uint_overflow.py

```diff
@@ -0,0 +1,123 @@
+from zkevm_specs.evm_circuit.util.call_gadget import CallGadget
+from ...util import (
+    FQ,
+    TxDataNonZeroGasEIP2028,
+    MAX_U64,
+    N_BYTES_U64,
+    TxGas,
+    TxGasContractCreation,
+    TxDataZeroGas,
+    # InitCodeWordGas,
+)
+from ..instruction import Instruction, Transition
+from ..table import CallContextFieldTag
+from ..opcode import Opcode
+
+
+def error_gas_uint_overflow(instruction: Instruction):
```

Comment:
We don't have to care about [`memoryCopierGas`](https://github.com/ethereum/go-ethereum/blob/793f0f9ec860f6f51e0cec943a268c10863097c7/core/vm/gas_table.go#LL65C6-L65C21) because it's handled by other error case.
It's used as gas calculation function [here](https://github.com/ethereum/go-ethereum/blob/793f0f9ec860f6f51e0cec943a268c10863097c7/core/vm/gas_table.go#L90).
And it's called through [dynamicGas](https://github.com/ethereum/go-ethereum/blob/793f0f9ec860f6f51e0cec943a268c10863097c7/core/vm/jump_table.go#L433) function.
The [`dynamicGas`](https://github.com/ethereum/go-ethereum/blob/793f0f9ec860f6f51e0cec943a268c10863097c7/core/vm/interpreter.go#L218) function error is handled as [`ErrorOutOfGas`](https://github.com/ethereum/go-ethereum/blob/793f0f9ec860f6f51e0cec943a268c10863097c7/core/vm/interpreter.go#L221).